index.html

```diff
@@ -843,6 +845,7 @@ <h1>Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="_self" href="https://twitter.com/repeatedly"><i class="icon social_twitter"></i></a>
 											<a href="http://jp.linkedin.com/in/repeatedly" target="_self"><i class="icon social_linkedin"></i></a>
+											<a href="https://m.facebook.com/people/Masahiro-Nakagawa/1251439284" target="_self"><i class="icon social_facebook"></i></a>
```

Comment:
Remove the mobile version facebook url

Reply:
What do you mean?

Reply:
@samyakbvs Provide the url for facebook website, rather than facebook mobile site